gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-inspector/policy-studio-debug-inspector-error/policy-studio-debug-inspector-error.component.scss

```diff
@@ -5,6 +5,7 @@
   }
 
   td {
+    width: 50%;
```

Comment:
I hadn't set the width so that the content on the right had more space. 
I think we could put longer, more explicit messages

Reply:
Or we could simply set the size of the first column to the minimum because we know the content. 
And perhaps change the text style to mark this column like a subtitle

Reply:
I changed it to be aligned with the condition table 🤷‍♂️